applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -468,6 +468,16 @@ Building and running
 
 .. include:: /includes/build_and_run.txt
 
+.. include:: ../../../samples/matter/lock/README.rst
+    :start-after: matter_door_lock_sample_nrf70_firmware_patch_start
+    :end-before: matter_door_lock_sample_nrf70_firmware_patch_end
+
+For example:
+
+   .. code-block:: console
+
+      west build -d nrf5340dk_nrf5340_cpuapp -d -- -DSHIELD=nrf7002ek -DCONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE=y -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3
+
```

Comment:
Do not add this information in the sample docs. This should be explained in the user guide in the configuration section (`nrf70_fw_patch_update.rst`) and just linked to here.

Reply:
In Matter samples I simplified that process, so we can skip some steps that are needed in a general guide.

Reply:
Proposed an edit to reduce the duplication of info with the guide.